File: python/2152.py
u = int(input())
i = 0

while i < u:
    log = input().split()
    print("{:0>2}:{:0>2} - A porta ".format(log[0],log[1]), end="")
    print("abriu!" if log[2] == "1" else "fechou!")
    i += 1

File: dart/2152.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync()!);

  for (int i = 0; i < n; i++) {
    List<String> o = stdin.readLineSync()!.split(" ");
    
    if (o[2] == "1") {
      print("${o[0].padLeft(2, "0")}:${o[1].padLeft(2, "0")} - A porta abriu!");
    } else {
      print("${o[0].padLeft(2, "0")}:${o[1].padLeft(2, "0")} - A porta fechou!");
    }
  }
}
